app/components/mentor/mentor.html

```diff
@@ -1,19 +1,33 @@
 <div class="body-container">
-  <section class="mentors">
-    <ul class="ch-grid" id="mentorsList" >
-      <li ng-repeat="mentor in mentors">
-        <div class="ch-item" style="background-image: url(../../../assets/images/img.png)">
-          <div class="ch-info">
-            <h3>{{mentor.first_name}} {{mentor.last_name}}</h3>
-            <p><a ng-click="goToDetails(mentor)">{{ "view_details" | translate }}</a></p>
-          </div>
-        </div>
-      </li>
-    </ul>
-    <ul style="list-style-type: none;">
-      <li ng-repeat="error in errors">
-        <span class="errorMessage">{{ error | translate }}</span>
-      </li>
-    </ul>
-  </section>
+  <div class="row">
+    <div class="content mentors-list">
+      <uib-tabset active="active">
+        <table class="table table-hover text-ellipsis">
+          <thead>
+            <tr>
+              <th class="col-lg-6 col-md-6 col-sm-6">{{ "last_name" | translate }}</th>
+              <th class="mentor-city">{{ "city" | translate }}</th>
+            </tr>
+          </thead>
+          <tbody>
+            <tr ng-repeat="mentor in mentors" ng-click="goToDetails(mentor)">
+              <td class="col-lg-6 col-md-6 col-sm-6 wrap-text ">
+                <ng-letter-avatar charCount = "2" height="35px" width="35px" fontSize="28px" avatarboard="true" avatarcustomborder="border:10px solid white" shape="round" data="{{mentor.first_name}} {{mentor.last_name}}"></ng-letter-avatar>
+                <span>{{ mentor.first_name }} {{mentor.last_name}}</span>
+              </td>
+              <td class="mentor-city">
+                <span class="glyphicon glyphicon-map-marker"></span>
+                <span>{{mentor.city}}</span>
+              </td>
+            </tr>
```

Comment:
why we don't have a view details or send invitation(context) button? as a organisation i can't create a context with this mentors.